qt/scientific_interfaces/Inelastic/Corrections/ApplyAbsorptionCorrections.cpp

```diff
@@ -19,6 +19,7 @@
 #include "MantidQtWidgets/Spectroscopy/SettingsWidget/SettingsHelper.h"
 
 #include <QStringList>
+#include <gmock/internal/gmock-internal-utils.h>
```

Comment:
Out of curiosity, why is this gmock include required inside this non-test file?

Reply:
This was unintended, I'm not sure how this include ended up there.